### PR TITLE
regiser user

### PR DESCRIPTION
**Pull request by TuTranVan**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      flash[:success] = "Regiser successfully!"
+      redirect_to root_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+    def user_params
```
Comment (trantrinh1102) [inline app/controllers/users_controller.rb]: indent

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  validates :name, presence: true, length: { maximum: 50 }
+  validates :phone, presence: true, length: { maximum: 10 }
+  validates :address, presence: true
+  validates :picture, presence: true
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 },
+                    format: { with: VALID_EMAIL_REGEX }, uniqueness: true
+
+  has_secure_password
```
Comment (trantrinh1102) [inline app/models/user.rb]: sắp xếp lại thứ tự
https://github.com/rubocop-hq/rails-style-guide#macro-style-methods

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  validates :password, length: { minimum: 6 }
+
+  private
+
+    def downcase_email
```
Comment (trantrinh1102) [inline app/models/user.rb]: indent

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Regiser successfully!"
+      redirect_to root_path
```
Comment (luongvietdung) [inline app/controllers/users_controller.rb]: `redirect_to root_path, success: "Regiser successfully!"`

**Pull request merged**